.github/dependabot.yml

```diff
@@ -60,3 +60,18 @@ updates:
   directory: "/"
   schedule:
     interval: daily
+
+- package-ecosystem: "gomod"
+  directory: "/examples/grpc-bridge/server"
+  schedule:
+    interval: daily
+
+- package-ecosystem: "gomod"
```

Comment:
this seems to be duplicated

also i think there are others afaict that we should include:

```console
$ find . -name go.mod
./examples/ext_authz/auth/grpc-service/go.mod
./examples/load-reporting-service/go.mod
./examples/grpc-bridge/server/kv/go.mod
./examples/grpc-bridge/server/go.mod

```